index.js

```diff
@@ -73,7 +73,8 @@ function getKinesisData(event, callback) {
 function filterGDEvents(cwEvents, callback) {
     async.filter(cwEvents,
         function(cwEvent, filterCallback){
-            return filterCallback(null, cwEvent.source && cwEvent.source == 'aws.guardduty');
+            return filterCallback(null, cwEvent.source && cwEvent.source 
```

Comment:
Could you please move the entire statement:
`cwEvent.source === 'aws.guardduty'`